packages/components/src/RadarChart/RadarChart.component.js

```diff
@@ -92,7 +92,7 @@ RadarChart.defaultProps = {
  */
 // TODO 6.0: remove this export
 export function LabelWithClick(props) {
-	const { activeAxis, className, index, textAnchor, payload, x, y } = props;
+	const { activeAxis, className, dataFeature, index, textAnchor, payload, x, y } = props;
```

Comment:
Could be nice to spread other props instead
```suggestion
	const { activeAxis, className, index, textAnchor, payload, x, y, ...rest } = props;
```